### PR TITLE
Reference: Get task by title

### PR DESCRIPTION
**Pull request by davkow**

### CODE REVIEW

File: docs/api/tasks-ref-topic-get-task-by-title.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+---
+layout: page
+---
+
+# Get task by title
+
+Returns an array of [`task`](task) objects only specified by the `title` parameter, if it exists.
```
Comment (rbwatson) [inline docs/api/tasks-ref-topic-get-task-by-title.md]:
```suggestion
Returns an array of [`task`](task) objects with the specified `title` parameter, if any exist.
```

File: docs/api/tasks-ref-topic-get-task-by-title.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+
+
+## URL
+
+```shell
+
+{GET}{server_url}/tasks/{title}
```
Comment (rbwatson) [inline docs/api/tasks-ref-topic-get-task-by-title.md]:
```suggestion
{GET} {server_url}/tasks?title={title}
```

Check to see if the syntax is correct for the query.

**Review by rbwatson — CHANGES_REQUESTED** [review]
Good work!
Review my comments and squash the commits, please.

Thanks!

File: docs/api/tasks-ref-topic-get-task-by-title.md (hunk 2 above)
Comment (davkow) [reply docs/api/tasks-ref-topic-get-task-by-title.md]: I checked. The syntax is correct.

**Comment by davkow** [comment]
@rbwatson: I can't squash the commits in GitHub Desktop until **after** I fix the "Resolve conflicts to continue..." with my-tutorial-topic branch. That is, I can't move to the my-reference-topic branch in GitHub Desktop without first fixing the conflict. 

I tried yesterday to squash commits using the CLI as you explained in the video. But I'm getting an error with VS code as my editor. I tried to research, but so far I haven't been able to fix. Another option is to change to vim as my editor and then do the squash steps, but I haven't tried that yet.

**Comment by davkow** [comment]
@rbwatson: The squash commits is done. Click on the Commits tab; you will see there is now only one commit.

**Review by rbwatson — APPROVED** [review]
LGTM
Rebased to main and ready to merge.

**Pull request merged**